### DISCUSSION TITLE
Two executables...CardinalJACK.exe and CardinalNative.exe (Which one?)

### DISCUSSION THREAD

**Discussion by 888visuals** [DISTRHO/Cardinal#601] [Q&A] [answered]
I have looked in the installation folder and the "docs" folder and there are no installation instructions. What is the difference between the two executables which are almost identical in size? Thanks!

**Comment by dromer** [comment, marked as answer]
CardinalJACK uses JACK audio backend: https://jackaudio.org/

CardinalNative uses your default windows soundcard (I believe).
